ci/recipe/meta.yaml

```
@@ -25,6 +25,8 @@ requirements:
     - biom-format >=2.1.5,<2.2.0
     - ijson
     - h5py
+    - matplotlib=3.1.0
```

Please use the conda package [match spec for guidelines](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications) here.

```suggestion
    - matplotlib 3.1.0
```

Please update `matplotlib-base` as appropriate, too.

---

Thank you, @thermokarst. :man_facepalming:
Should be in better shape now.